verify-loader

```diff
@@ -135,16 +137,21 @@ def verify(input_gen, report_interval=REPORT_INTERVAL):
     try:
         for line in input_gen:
             line_len = len(line)
-            if not line.startswith("loader seq - "):
+            #find the log header: loader seq -
+            #for container logs the line will not start with header - instead there's a timestamp; if header not present ignore this line 
+            if "loader seq - " not in line:
                 report_ignored_bytes += line_len
                 report_ignored_count += 1
             else:
+                #check that line read has constituent parts after header (<header> - <uuid> - <seq_num> - <payload>)
                 try:
-                    _, invocid, seqval, payload = line.split('-', 4)
+                    indx = line.find('loader seq -') 
```

Comment:
Same as above, we don't want to add support for prefixed data of the log lines generated by the `loader`.  If there is a prefix, let's have another tool strip it before sending the data to `verify-loader`.